src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1383,8 +1560,11 @@ static void fill_idPtypeP
   bgP->boPP.append(typePatternQ, true);
   bgP->boPP.append(CSUB_EXPIRATION, bobGtCurrentTime.obj());
   bgP->boPP.append(CSUB_STATUS, bobNeStatus.obj());
+#if 0
   bgP->boPP.appendCode("$where", bgP->functionIdPtypeP);
-
+#else
+  bgP->boPNP.append("$expr", outer_obj.obj());
```

Comment:
```suggestion
  bgP->boPP.append("$expr", outer_obj.obj());
```

This probably fix the tests that are currently broken.